common/include/villas/kernel/devices/device.hpp

```diff
@@ -0,0 +1,33 @@
+/* Interface for Linux/Unix devices.
+ *
+ * Author: Pascal Bauer <pascal.bauer@rwth-aachen.de>
+ *
+ * SPDX-FileCopyrightText: 2023-2024 Pascal Bauer <pascal.bauer@rwth-aachen.de>
```

Comment:
Just wondering: has this work been done as part of a contract with RWTH? Shouldnt we add RWTH as the copyright holder in this case?

Reply:
I felt like the result of our discussion on this topic was that we do not enforce whether we list a company or a person here. I'm still of the opinion that the university cannot be the copyright owner.

Reply:
Okay, I am giving up :D

Reply:
Little background on why I am so persistent about this topic:

We had several cases in the past in which software has been relicensed.
Doing so requires the consent from all contributors (or their companies which hold the rights on their contributions). With contributions like the one here, its impossible to distinguish whether it has been made by an individual or somebody employed.

So in case of any copyright related questions we will always have to reach our to the persond directly. And you know that this can be difficult at times for students who have long left the institute.